README.md

```diff
@@ -42,6 +42,33 @@ Or if the log file is stored locally:
+[error: Bad exit status from /var/tmp/rpm-tmp.8C0L25 (%check)] : An error message is reported indicating that the 'check' phase of the rpm build process failed with a bad exit status.
+[EXCEPTION: [Error('Command failed: \n # /usr/bin/systemd-nspawn -q -M 9e5f3eacb5dc4578ac1388b5dbdbf253 -D /var/lib/mock/f40-flatpak-app-build-5101 6792-6091603/root -a -u mockbuild --capability=cap_ipc_lock --bind=/tmp/mock-resolv.x4datb56:/etc/resolv.conf --bind=/dev/btrfs-control --bind=/dev/mapper/control --bind=/dev/fuse --bind=/dev/loop-control --bind=/dev/loop0 --bind=/dev/loop1 --bind=/dev/loop2 --bind=/dev/loop3 --bind=/dev/loop4 --bind=/dev/loop5 --bind=/dev/loop6 --bind=/dev/loop7 --bind=/dev/loop8 --bind=/dev/loop9 --bind=/dev/loop10 --bind=/dev/loop11 --console=pipe --setenv=TERM=vt100 --setenv=SHELL=/bin/bash --setenv=HOME=/builddir --setenv=HOSTNAME=mock --setenv=PATH=/usr/bin:/bin:/usr/sbin:/sbin '--setenv=PROMPT_COMMAND=printf "\033]0;<mock-chroot>\007"' '--setenv=PS1=<mock-chroot> \s-\v\$ ' --setenv=LANG=C.UTF-8 --resolv-conf=off bash --login -c '/usr/bin/rpmbuild -bb --noclean --target noarch --nodeps /builddir/build/SPECS/python-autobahn.spec'\n', 1)] : The traceback shows that the error is due to a command executed during the rpm build process returning a non-zero exit status. The command in question is a systemd-nspawn command used to ru n the rpmbuild command inside a mock chroot environment. The error message indicates that the command failed.
+[mockbuild.exception.Error: Command failed: \n # /usr/bin/systemd-nspawn -q -M 9e5f3eacb5dc4578ac1388b5dbdbf253 -D /var/lib/mock/f40-flatpak-app-build-51016792-6091603/root -a -u mockbuild --capability=cap_ipc_lock --bind=/tmp/mock-resolv.x4datb56:/etc/resolv.conf --bind=/dev/btrfs-control --bind=/dev/mapper/control --bind=/dev/fuse --bind=/dev/loop-control --bind=/dev/loop0 --bind=/dev/loop1 --bind=/dev/loop2 --bind=/dev/loop3 --bind=/dev/loop4 --bind=/dev/loop5 --bind=/dev/loop6 --bind=/dev/loop7 --bind=/dev/loop8 --bind=/dev/loop9 --bind=/dev/loop10 --bind=/dev/loop11 --console=pipe --setenv=TERM=vt100 --setenv=SHELL=/bin/bash --setenv=HOME=/builddir --setenv=HOSTNAME=mock --setenv=PATH=/usr/bin:/bin:/usr/sbin:/sbin \'--setenv=PROMPT_COMMAND=printf "\033]0;<mock-chroot>\007"' '--setenv=PS1=<mock-chroot> \s-\v\$ ' --setenv=LANG=C.UTF-8 --resolv-conf=off bash --login -c '/usr/bin/rpmbuild -bb --noclean --target noarch --nodeps /builddir/build/SPECS/python-autobahn.spec' ] : This is the detailed traceback of the error, showing the exact command that failed and the exit status. It is clear that the rpmbuild command inside the systemd-nspawn command failed with a non-zero exit status. The reason for the failure is not provided in the log snippet. However, the Pytest warnings do not appear to be related to the rpm build error.
+```
```

Comment:
The lines are really loong. What about wrapping them? https://stackoverflow.com/questions/41238148/how-to-wrap-long-lines-inside-of-markdown-code-in-github-and-gitlab-issu

Reply:
thanks for the suggestion, will wrap